From 7e982e4a89d1f2e23b121c2db5386dc348f0e52b Mon Sep 17 00:00:00 2001
From: Lu jicong <jiconglu58@gmail.com>
Date: Wed, 29 Jan 2025 19:46:03 +0800
Subject: [PATCH] nanopc-t4: use 1600MHz sdram configuration

Some boards are unstable at 1866MHz configuration
Switch to 1600MHz to avoid unstable problems
This is also consistent with stock frequency
---
 arch/arm/dts/rk3399-nanopc-t4-u-boot.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/rk3399-nanopc-t4-u-boot.dtsi b/arch/arm/dts/rk3399-nanopc-t4-u-boot.dtsi
index 17201bcf41..8b6c9059ab 100644
--- a/arch/arm/dts/rk3399-nanopc-t4-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-nanopc-t4-u-boot.dtsi
@@ -4,4 +4,4 @@
  */
 
 #include "rk3399-nanopi4-u-boot.dtsi"
-#include "rk3399-sdram-lpddr3-samsung-4GB-1866.dtsi"
+#include "rk3399-sdram-lpddr3-4GB-1600.dtsi"
-- 
2.39.5

